cypress/test/tools/Filters.cy.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { mount } from 'cypress/react';
+import Filters from '../../../components/tools/Filters';
+import ToolFilter from '../../../context/ToolFilterContext';
+import { RouterContext } from 'next/dist/shared/lib/router-context';
+
+describe('Filters Component', () => {
+  //filter types and their data-testid attributes
+  const filterTypes = [
+    { name: 'Language', id: 'Filters-Language-dropdown' },
+    { name: 'Technology', id: 'Filters-Technology-dropdown' },
+    { name: 'Category', id: 'Filters-Category-dropdown' },
+  ];
+
+  // select from dropwdown
+  const selectFilters = (filterType) => {
+    cy.get(`[data-testid="${ filterType.id }"]`).click();
+  };
+
+  // helper function to verify the applied filters
+  const verifyFilters = (filterType) => {
+    // check that applied filters contain the selected filters from the dropdown
```

Comment:
Where are you verifying the filters using context variables?

Reply:
I am not using context variables to verify .